builder/rootfs/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN apk add --update-cache \
     && rm -rf /var/cache/apk/*
 
 # the docker package in alpine disables aufs and devicemapper
-RUN curl -sSL https://get.docker.com/builds/Linux/x86_64/docker-1.5.0 -o /usr/bin/docker && \
+RUN curl -sSL https://get.docker.com/builds/Linux/x86_64/docker-1.8.3 -o /usr/bin/docker && \
```

Comment:
I think it's possible this could lead to the "simultaneous docker pull" bug, so I will investigate adding the equivalent of a `flock` guard to the relevant builder code.